### PR TITLE
API fields, fixes and upgrades

### PR DESCRIPTION
**Pull request by alejandrod**
Adding missing fields and fixing according to docs.
Upgrading related libraries

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -54,7 +54,6 @@
 	</dependencies>
 
 	<build>
-		<finalName>wepay</finalName>
```
Comment (alejandrod) [inline pom.xml]: It's a bad idea to have a fixed name

File: src/main/java/com/wepay/model/data/AddressData.java [hunk 2]
```diff
@@ -1,40 +1,28 @@
...
+  public String country;
 
-	public String address1;
-	public String address2;
-	public String city;
-	public String state;
-	public String zip;
-	public String country;
-	public String region;
-	public String postcode;
```
Comment (alejandrod) [inline src/main/java/com/wepay/model/data/AddressData.java]: Not sure where postcode is used. Based on the API docs is postal_code

File: src/main/java/com/wepay/model/data/HeaderData.java [hunk 3]
```diff
@@ -4,4 +4,7 @@
     public String accessToken;
     public String riskToken;
     public String clientIP;
+
```
Comment (alejandrod) [inline src/main/java/com/wepay/model/data/HeaderData.java]: Allows to set specific timeouts per request if needed

**Comment by snznedta** [comment]
@alejandrod  Thanks for contributing! 
General comments:
1.please resolve conflict.
2.please follow the coding style (indents) that we currently have.

File: src/main/java/com/wepay/model/data/PhoneData.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package com.wepay.model.data;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class PhoneData {
```
Comment (snznedta) [inline src/main/java/com/wepay/model/data/PhoneData.java]: Looks like duplicated as `InternationalPhoneNumberData.java`
please replace it.

File: src/main/java/com/wepay/model/data/AddressData.java [hunk 5]
```diff
@@ -1,40 +1,28 @@
...
-		JSONObject o = new JSONObject();
-		o.put("address1", info.address1);
-		o.put("address2", info.address2);
-		o.put("city", info.city);
-		o.put("region", (info.state != null) ? info.state : info.region);
-		o.put("postal_code", (info.zip  != null) ? info.zip : info.postcode);
-		o.put("country", info.country);
-		return o;
-	}
+  public AddressData() {
```
Comment (snznedta) [inline src/main/java/com/wepay/model/data/AddressData.java]: Unnecessary constructor here.

File: src/main/java/com/wepay/model/data/HeaderData.java (hunk 3 above)
Comment (snznedta) [reply src/main/java/com/wepay/model/data/HeaderData.java]: We have 60s timeout from server side but received several requests from contributors to make client-side timeout configurable. Our team is still having discussion internally and will modify it after we achieving a conclusion. So let's hold off making client-side timeout configurable currently.

File: src/main/java/com/wepay/model/data/SsoTokenData.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+package com.wepay.model.data;
+
+public class SsoTokenData {
```
Comment (snznedta) [inline src/main/java/com/wepay/model/data/SsoTokenData.java]: I prefer to use `SSO` instead of `Sso`

**Comment by alejandrod** [comment]
@snznedta Hey, Looks like someone has already put in some of my changes. That's great.
There are a few pending. Can we merge?
Regarding formatting. I'd strongly recommend to merge and after to a commit with only formatting changes. Is super easy to format all the classes.
There is a mix of spaces and tabs, 2 vs 4 spaces, etc. I'd recommend not to use tabs at all. 
In any case, I'm pretty sure that using your preferred UI you can format them all with your preferred settings.

**Comment by y2chaits** [comment]
Hi @alejandrod , we're planning to make some of the fixes you suggested in a general way across the entire sdk. We're also still discussing your other changes internally, and will get back to you when we know more. Thanks for contributing! we appreciate it!

**Comment by SharonLingqiongTan** [comment]
Hi @alejandrod , the pull request looks good to me. I'm going to merge it.

**Pull request merged**